packages/components/src/components/button/button.tsx

```diff
@@ -61,6 +61,10 @@ export class Button {
   @Prop() download?: string;
   /** (optional) Set `tabindex` in the inner button or link element */
   @Prop() innerTabindex?: number;
+  /** (optional) Button aria-label */
+  @Prop() ariaLabel?: string;
+  /** (optional) Button form */
+  @Prop() form?: string;
```

Comment:
Regarding the `form` prop, it should work as it is now, but I do have 2 suggestions:

- we have [a couple](https://github.com/telekom/scale/blob/ee568c01e482508f1835f6f6d46b9595d86822b5/packages/components/src/components/button/button.tsx#L133) [of "hacks"](https://github.com/telekom/scale/blob/ee568c01e482508f1835f6f6d46b9595d86822b5/packages/components/src/components/button/button.tsx#L96) in place to deal with forms already. In them we do `this.hostElement.closest('form')` to query for a form. If you use the `form` attribute, probably the button is not a direct child of the relevant form, so we would need to query for that `id` instead of just a <form> if the `form` prop is present… to make sure the hack works as expected…
- also, if we add the `form` prop, I think it would make sense to also add the rest of the `form-` attributes a button can take — in which case I think they would need to be copied over to the [hack button](https://github.com/telekom/scale/blob/ee568c01e482508f1835f6f6d46b9595d86822b5/packages/components/src/components/button/button.tsx#L104-L106) too.
  - [formaction](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-formaction)
  - [formenctype](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-formenctype)
  - [formmethod](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-formmethod)
  - and so on…

Reply:
Thanks for the tip 🙌🏻, I would be in favor of removing the form property. Instead I will offer an alternative to the issue creator.